### PR TITLE
Add scenario for user-specified key

### PR DESCRIPTION
**Pull request by mnm678**
This moves scenario 10 from #66 into it's own pr with a couple of minor clarifications.

### CODE REVIEW

**Comment by sudo-bmitch** [comment]
LGTM

**Comment by mnm678** [comment]
@SteveLasker or @justincormack I don't have permission to merge in this repo, could you add this and #47?

**Comment by dlorenc** [comment]
Lgtm as well!

File: scenarios.md [hunk 1]
```diff
@@ -250,6 +250,18 @@ A weakness is discovered in a widely used cryptographic algorithm and a decision
...
+### Scenario #10: Specifying a trusted key
+
+A user may want to trust an artifact only if it is signed by a specific key. Maybe this is an internally created artifact with a known signing key. This key may be distributed using a trusted third party mechanism.
+
+1. The user obtains a trusted key for a particular artifact.
+1. The user downloads and verifies the artifact using Notary v2 and their known key.
+
+**Implications of this requirement**
+
+1. Users must be allowed to configure specific trusted keys for specific artifacts or collections of artifacts.
```
Comment (SteveLasker) [inline scenarios.md:263]: The wording around collections of artifacts might be inferred to something around a repository collection. As artifacts move within and across registries, a repository may have multiple signing keys. 
I'll LGTM and we can clarify later if needed.

**Review by SteveLasker — APPROVED** [review]
LGTM with minor question, to be clarified later if needed, around collections of artifacts.

**Pull request merged**